backend/src/main/java/ee/tlu/forum/service/ThreadServiceInterface.java

```diff
@@ -6,9 +6,9 @@
 import java.util.List;
 
 public interface ThreadServiceInterface {
-    Thread createThread(AddNewThreadInput form);
+    Thread createThread(AddNewThreadInput form, String token);
     void deleteThreadById(Long id);
```

Comment:
Does this also need a token?

Reply:
Yes, as admins and users have different permissions but can access the same endpoint